images/base/Dockerfile.centos7

```diff
@@ -4,9 +4,9 @@
 #
```

Comment:
At this point I am confident that if `images/base/Dockerfile` works for you, we should merge them and remove the CentOS one (and all of the others)